scripts/gentest/src/main.rs

```diff
@@ -203,18 +210,95 @@ fn generate_assertions(ident: &str, node: &json::JsonValue) -> TokenStream {
     )
 }
 
+// type QuoteProp = Fn(prop_name: &str, quoter: impl Fn(&json::object::Object) -> TokenStream) -> TokenStream;
```

Comment:
Is this something we like to keep around? If yes, please add a comment explaining why, if no then remove it?

Reply:
```suggestion
```